_data/simple-icons.json

```
@@ -1720,6 +1720,12 @@
             "hex": "F7901E",
             "source": "https://www.boostmobile.com"
         },
+        {
+            "title": "Boosty",
+            "hex": "F15F2C",
+            "source": "https://boosty.to/app/brand?locale=en_US",
+            "guidelines": "https://boosty.to/app/brand?locale=en_US#:~:text=PNG-,Logo%20use%20rules,-Failure%20to%20comply"
+        },
```

```suggestion
        {
            "title": "Boosty",
            "hex": "F15F2C",
            "source": "https://boosty.to/app/brand",
            "guidelines": "https://boosty.to/app/brand"
        },
```

Those queries need to be removed.